gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"github.com/gluster/gluster-prometheus/pkg/glusterutils"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	volumeLabels = []string{
+		"volume",
+	}
+
+	glusterVolumeTotalCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_total_count",
+			Help:      "Total no of volumes",
+		},
+		[]string{},
+	)
+	glusterVolumeCreatedCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_created_count",
+			Help:      "Freshly created no of volumes",
+		},
+		[]string{},
+	)
+	glusterVolumeStartedCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_started_count",
+			Help:      "Total no of started volumes",
+		},
+		[]string{},
+	)
+	glusterVolumeBrickCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_brick_count",
+			Help:      "Total no of bricks in volume",
+		},
+		volumeLabels,
+	)
+)
+
+func getVolumeLabels(volname string) prometheus.Labels {
+	return prometheus.Labels{
+		"volume": volname,
+	}
+}
+
+func VolumeCounts() {
+	volumes, _ := glusterutils.VolumeInfo(&glusterConfig)
+
+	volCount, volStartCount, volCreatedCount := 0, 0, 0
```

Comment:
`var volCount, volStartCount, volCreatedCount int` will initializes to 0

Reply:
Ack